src/koopmans/io/_kwf.py

```diff
@@ -50,6 +67,22 @@ def object_hook(dct):
         module = import_module(subdct['__module__'])
         return getattr(module, subdct['__name__'])
     else:
+        if ('scaler' in dct or 'model' in 'dct'):
+            def load_ml_model(model, model_dct):
+                model.set_params(**model_dct['init_params'])
+                for p in model_dct['model_params'].keys():
+                    setattr(model, p, np.asarray(model_dct['model_params'][p] ))
+                return model
+            if 'scaler' in dct:
+                scaler = StandardScaler()
+                model_dct = dct.pop('scaler')
+                scaler = load_ml_model(scaler, model_dct)
+                dct['scaler'] = scaler
+            if 'model' in dct:
+                model = Ridge()
+                model_dct = dct.pop('model')
+                model = load_ml_model(model, model_dct)
+                dct['model'] = model
```

Comment:
Unless there is a good reason not to, let's not place this inside of the `else` statement but instead use the pattern above for `Path` i.e. initially create the dct inside of `KoopmansEncoder` with a `__standardscaler__` or `__ridge__` entry, and then do something similar to line 64